public_html/wp-content/plugins/wcpt/wcpt-meetup/class-meetup-application.php

```diff
@@ -228,11 +234,9 @@ public function create_post( $data ) {
 		add_post_meta( $post_id, 'Address', $organizer_address );
 		add_post_meta( $post_id, 'Already a meetup', $data['q_already_a_meetup'] );
 		add_post_meta( $post_id, 'Meetup URL', $data['q_existing_meetup_url'] );
-		add_post_meta( $post_id, 'Organizer description', $data['q_describe_yourself'] );
 		add_post_meta( $post_id, 'Primary organizer WordPress.org username', $data['q_wporg_username'] );
 		add_post_meta( $post_id, 'Slack', $data['q_wp_slack_username'] );
 		add_post_meta( $post_id, 'Date Applied', time() );
-		add_post_meta( $post_id, 'Extra Comments', $data['q_additional_comments'] );
```

Comment:
Same question - should this save the content from `q_anything_else`?

Reply:
Same answer than above :)